pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/defaultcolumn/BaseDefaultColumnHandler.java

```diff
@@ -363,6 +379,12 @@ protected boolean createColumnV1Indices(String column)
                   argument);
               return false;
             }
+            // TODO: Support creation of derived columns from forward index disabled columns
+            if (!_segmentWriter.hasIndexFor(argument, ColumnIndexType.FORWARD_INDEX)) {
+              LOGGER.warn("Skip creating derived column: {} because argument: {} does not have a forward index", column,
```

Comment:
Throwing error is better to force the user to not create a derived column from noForwardIndex column until reload code path supports everything (including derived column handling possibly) ?

Reply:
done